lib/envm/env_var.rb

```diff
@@ -17,16 +18,20 @@ def initialize(name:, description: nil, default_value: nil, required: [])
       end
     end
 
+    def required_and_missing?
+      required_environments.include?(Config.environment) && !system_value
+    end
```

Comment:
I think I would prefer if we broke this up into two different methods: `#required?` and `#missing?`